cardinal/option.go

```diff
@@ -82,3 +82,9 @@ func WithPrettyLog() WorldOption {
 		ecsOption: ecs.WithPrettyLog(),
 	}
 }
+
+func WithCORS() WorldOption {
+	return WorldOption{
+		serverOption: nil,
+	}
+}
```

Comment:
something missing here 🕵🏻‍♂️

Reply:
eh good catch.

Reply:
fixed added coverage for this function too.